language/de_x_sie/acp/extensions.php

```diff
@@ -71,6 +71,12 @@
 	'EXTENSION_NAME'			=> 'Name der Erweiterung',
 	'EXTENSION_ACTIONS'			=> 'Vorgänge',
 	'EXTENSION_OPTIONS'			=> 'Optionen',
+	'EXTENSION_INSTALL_HEADLINE'=> 'Eine Erweiterung installieren',
+	'EXTENSION_INSTALL_EXPLAIN'	=> '<ol>
+			<li>Laden Sie die Erweiterung aus der phpBB-Erweiterungsdatenbank herunter</li>
+			<li>Entpacken Sie die Erweiterung und laden Sie sie in das <samp>ext/</samp>-Verzeichnis Ihrer phpBB-Installation hoch.</li>
```

Comment:
In "en" haben wir da keinen Punkt am Ende. 

Also entweder alle drei mit Punkt oder keine, wie in "en".